app/models/service.rb

```diff
@@ -38,6 +38,8 @@ class Service < ApplicationRecord
   before_destroy :stop_destroy_if_last_or_default
   after_destroy :update_account_default_service
 
+  after_commit :achieve_as_deleted, on: :destroy
```

Comment:
`achieve` ? what does it mean? :D

Reply:
hahahahah it should be `archieve` as I see from `gmail` 😊

Reply:
i will fix it in another PR because I have to fix it for other classes too 😊

Reply:
then `archive`